community-operators/jaeger/jaeger.v1.8.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: jaeger-operator.v1.8.2
+  namespace: placeholder
+  annotations:
+    categories: "tracing, monitoring, troubleshooting, distributed"
+    certified: "false"
+    containerImage: docker.io/jaegertracing/jaeger-operator:1.8.2
+    createdAt:  2019-01-09T12:00:00Z
+    support: Jaeger
+spec:
+  displayName: jaeger-operator
+  description: |-
+    Provides monitoring and troubleshooting microservices-based distributed systems
+  keywords: ['tracing', 'monitoring', 'troubleshooting', 'distributed']
+  version: 1.8.2
+  maintainers:
+  - name: Jaeger Google Group
+    email: jaeger-tracing@googlegroups.com
+  provider:
+    name: Jaeger
+  labels:
+    name: jaeger-operator
+  selector:
+    matchLabels:
+      name: jaeger-operator
+  links:
+  - name: Jaeger Operator Source Code
+    url: https://github.com/jaegertracing/jaeger-operator
+  installModes:
+  - type: OwnNamespace
+    supported: false
+  - type: SingleNamespace
+    supported: false
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
```

Comment:
I believe this operator can be configured to watch a single namespace, own namespace or all namespaces. https://github.com/jaegertracing/jaeger-operator/blob/master/pkg/cmd/start/main.go#L84. So, I think we need to implement `installModes` for this operator.